app/views/chargeback/_cb_rate_edit.html.haml

```diff
@@ -1,19 +1,20 @@
 - url = url_for(:action => 'cb_rate_form_field_changed', :id => @edit[:rec_id] || "new")
 - currency = ChargebackRateDetailCurrency.currencies_for_select
 #form_div
-  %h3
-    = _('Basic Info')
-  .form-horizontal.static
-    .form-group
-      %label.col-md-2.control-label
-        = _('Description')
-      .col-md-8
-        = text_field_tag("description", @edit[:new][:description],
-                        :maxlength => MAX_NAME_LEN, "data-miq_observe" => {:interval => '.5', :url => url}.to_json,
-                        :class => "form-control")
-        = javascript_tag(javascript_focus('description'))
+  - if (@edit[:new][:description] != "Default Container Image Rate")
```

Comment:
I actually don't know the answer to the `:default => true` problem.

At the very least, the test for whether or not should the name be editable should be
in a controller, not a view.

Also, just hiding the form field isn't enough. Same test needs to be done on form submission
side.